--- libpqxx-2.6.6/configure.ac.in.orig	2006-05-27 10:44:40.000000000 +0200
+++ libpqxx-2.6.6/configure.ac.in	2006-05-28 17:39:28.000000000 +0200
@@ -247,7 +247,7 @@
 
 AC_SUBST(with_postgres_lib)
 AC_SUBST(with_postgres_include)
-POSTGRES_LIB="-R${with_postgres_lib}"
+POSTGRES_LIB=
 AC_SUBST(POSTGRES_LIB)
 POSTGRES_INCLUDE="-I${with_postgres_include}"
 AC_SUBST(POSTGRES_INCLUDE)
--- libpqxx-2.6.6/configure.ac.orig	2006-05-27 12:46:32.000000000 +0200
+++ libpqxx-2.6.6/configure.ac	2006-05-28 17:39:34.000000000 +0200
@@ -247,7 +247,7 @@
 
 AC_SUBST(with_postgres_lib)
 AC_SUBST(with_postgres_include)
-POSTGRES_LIB="-R${with_postgres_lib}"
+POSTGRES_LIB=
 AC_SUBST(POSTGRES_LIB)
 POSTGRES_INCLUDE="-I${with_postgres_include}"
 AC_SUBST(POSTGRES_INCLUDE)
